jvcl/run/JvDSADialogs.pas

```diff
@@ -850,8 +850,16 @@ function CreateDSAMessageForm(const ACaption, Msg: string; const APicture: TGrap
     CenterParLeft := 0;
     CenterParTop := 0;
     {$ifdef RTL210_UP}
-    CenterParWidth := TScreen(CenterParent).MonitorFromWindow(TScreen(CenterParent).ActiveCustomForm.Handle).Width;
-    CenterParHeight := TScreen(CenterParent).MonitorFromWindow(TScreen(CenterParent).ActiveCustomForm.Handle).Height;
+    if  TScreen(CenterParent).ActiveCustomForm <> nil  then
```

Comment:
`TScreen(CenterParent).ActiveCustomForm` should be stored in a local variable and then the test should be written like this:
`if Assigned(CenterParentActiveCustomForm) then` 
note the single space BEFORE and AFTER the test, I have no idea where you got the idea of using two spaces on each side.
Once there is a local variable, it should also be used on the two other lines

Reply:
yes, I do believe that not repeating `TScreen(CenterParent).ActiveCustomForm` everywhere will improve readability by shortening already too long rows. If this was targeting really recent versions of Delphi, I would even go as far as using inline variables.